RealmSwift/Tests/TestCase.swift

```diff
@@ -113,6 +113,24 @@ class TestCase: XCTestCase {
         queue.sync { }
     }
 
+    /// Check whether two test objects are equal (refer to the same row in the same Realm), even if their models
+    /// don't define a primary key.
+    func assertEqual(_ o1: Object?, _ o2: Object?, fileName: String = #file, lineNumber: UInt = #line) {
+        RLMAssertEqualTestObjects(self, o1, o2, fileName, lineNumber)
```

Comment:
Couldn't this be implemented in terms of `Object.isEqual(to:)`, rather than calling into Obj-C and using `RLMObjectBaseAreEqual`?